packages/webapp/pages/jobs/[id]/questions.tsx

```diff
@@ -74,22 +86,60 @@ const AcceptPage = (): ReactElement => {
   const {
     file,
     setFile,
-    handleUpload,
+    handleUpload: handleUploadCV,
     isPending: isUploadPending,
   } = useCVUploadManager(acceptOpportunity);
 
+  const handleUpload = useCallback(() => {
+    logEvent({
+      event_name: LogEvent.AnswerScreeningQuestion,
+      target_id: opportunityId,
+      extra: JSON.stringify({
+        question_type: 'cv',
```

Comment:
@idoshamun I added this question_type here so we can keep track of the newly added types as well